pkg/stanza/operator/input/windows/api.go

```diff
@@ -38,6 +47,10 @@ const (
 	EvtSubscribeStartAtOldestRecord uint32 = 2
 	// EvtSubscribeStartAfterBookmark is a flag that will subscribe to all events that begin after a bookmark.
 	EvtSubscribeStartAfterBookmark uint32 = 3
+	// EvtRpcLoginClass is a flag that indicates the login class.
+	EvtRpcLoginClass uint32 = 1
+	// EvtSubscribeOriginMask is a flag that indicates the origin mask.
+	EvtSubscribeActionDeliver = 1
```

Comment:
I don't think this is being used in this change.

Reply:
removed unused action